code/style.css

```diff
@@ -1,58 +1,233 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
   font-family: "Courier New", Courier, monospace;
 }
 
-p {
+.wrap {
+  padding: 20px;
+  background-color: #c44536;
+}
+
+.header {
+  display: flex;
+  justify-content: center;
+  justify-content: space-around;
+  align-items: center;
+}
+
+.slogan {
+  text-align: center;
+  font-size: 45px;
+  padding: 10px;
+  color: white;
+  font-family: 'Ranchers', cursive;
+  background-color: #c44536;
+}
+
+.cardContent {
   margin: 1px;
   font-weight: 300;
   background-color: #fff;
 }
 
+.killerIcon {
+  margin-top: 60px;
+  width: 50%;
+  fill: rgba(255,255,255,0.5);
+}
+
+.killerIcon.hide {
+  display: none;
+}
+
+.weaponIcon {
+  margin-top: 60px;
+  width: 60%;
+  fill: rgba(255,255,255,0.3);
+}
+
+.roomIcon {
+  margin-top: 70px;
+  width: 70%;
+  fill: rgba(255,255,255,0.3);
+}
+
+.deckTitle {
+  background-color: #8d0801;
+  color: white;
+  font-weight: bold;
+  font-size: 20px;
+  border-top: 1px solid black;
+  border-bottom: 1px solid black;
+}
+
 .cards {
-  margin: 50px auto;
-  width: 90%;
-  height: 600px;
-  background: #ccc;
-  display: flex;
-  justify-content: space-around;
+  margin: 30px;
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
   align-items: center;
+  justify-items: center;
 }
 
-.killer-deck {
-  width: 250px;
-  height: 400px;
-  background: #000;
+.deck {
```

Comment:
I really like that you have styled all cards in one. =)

It would be great with a comment here to tell that you are merging the three different cards. Or, if you would have used kebab-case when naming the classes you could have written this style as: 
.deck-killer-card, .deck-weapon-card, .deck-room-card {
}